modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/utils/constants/autoCloseAlertTime.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * The contents of this file are subject to the terms of the Liferay Enterprise
+ * Subscription License ("License"). You may not use this file except in
+ * compliance with the License. You can obtain a copy of the License by
+ * contacting Liferay, Inc. See the License for the specific language governing
+ * permissions and limitations under the License, including but not limited to
+ * distribution rights of the Software.
+ */
+
+export const AUTO_CLOSE_ALERT_TIME = {
+	fail: false,
+	success: 2000,
+};
```

Comment:
I think this object would be better to be created in the generic Toast component file. (As I asked to create this component before). The reason is because this is very specific to the toast, so we can move to the same file.

Reply:
Yeah, make senses. But in CP all download alert should have same time:

fail: failse,
success: 2000,

What do you think change the const name to something like 'AUTO_CLOSE_DOWNLOAD_ALERT_TIME'?

Reply:
You're right! We can keep the file where is it. 
Yeah, AUTO_CLOSE_DOWNLOAD_ALERT_TIME seems good!